static/templates/more_topics_spinner.hbs

```diff
@@ -1,3 +1,3 @@
 <li class="searching-for-more-topics">
-    <img src="/static/images/loading/loading-ellipsis.svg" alt="" />
+    <img src="{{ loading_image }}" alt="" />
 </li>
```

Comment:
I feel like this patterns feels bad for the readability of these templates -- like in an ideal world we'd configure Handlebars to just do it. given it's only a half-dozen templates, I'm fine with integrating the change this way. The main thing that's unpleasant about it is that AFAIK we won't get any error if code rendering one of these templates fails to pass one of the variables.

Reply:
Yeah, agree.  Repushed with a better solution.